### PR TITLE
Add description and contact_info columns to network tables

### PR DESCRIPTION
**Pull request by ajmccarthy**
Add description and contact_info columns to network and network_staging tables.  Uses addColumn changesets.

### CODE REVIEW

File: liquibase/changeLogs/nwis/tables/network/changeLog.yml [hunk 1]
```diff
@@ -19,4 +19,33 @@ databaseChangeLog:
...
             relativeToChangelogFile: true
-        - rollback: drop table if exists ${NWIS_SCHEMA_NAME}.network cascade;
\ No newline at end of file
+        - rollback: drop table if exists ${NWIS_SCHEMA_NAME}.network cascade;
+
+  - changeSet:
+      author: ajmccart
+      id: "add.description_column.${NWIS_SCHEMA_NAME}.network"
+      preConditions:
+        - onFail: MARK_RAN
```
Comment (dsteinich) [inline liquibase/changeLogs/nwis/tables/network/changeLog.yml]: This should probably HALT - we need to stop the run and fix something if the table we are adding the column to does not exist.

File: liquibase/changeLogs/nwis/tables/network/changeLog.yml [hunk 2]
```diff
@@ -19,4 +19,33 @@ databaseChangeLog:
...
+              tableName: network
+      changes:
+        - sql : alter table ${NWIS_SCHEMA_NAME}.network add column if not exists description text;
+        - rollback: alter table ${NWIS_SCHEMA_NAME}.network drop column description;
+        
+  - changeSet:
+      author: ajmccart
+      id: "add.contactinfo_column.${NWIS_SCHEMA_NAME}.network"
+      preConditions:
+        - onFail: MARK_RAN
```
Comment (dsteinich) [inline liquibase/changeLogs/nwis/tables/network/changeLog.yml]: Also HALT

File: liquibase/changeLogs/nwis/tables/networkStaging/changeLog.yml [hunk 3]
```diff
@@ -19,4 +19,33 @@ databaseChangeLog:
...
             relativeToChangelogFile: true
-        - rollback: drop table if exists ${NWIS_SCHEMA_NAME}.network_staging cascade;
\ No newline at end of file
+        - rollback: drop table if exists ${NWIS_SCHEMA_NAME}.network_staging cascade;
+        
+  - changeSet:
+      author: ajmccart
+      id: "add.description_column.${NWIS_SCHEMA_NAME}.network_staging"
+      preConditions:
+        - onFail: MARK_RAN
```
Comment (dsteinich) [inline liquibase/changeLogs/nwis/tables/networkStaging/changeLog.yml]: HALT

File: liquibase/changeLogs/nwis/tables/networkStaging/changeLog.yml [hunk 4]
```diff
@@ -19,4 +19,33 @@ databaseChangeLog:
...
+              tableName: network_staging
+      changes:
+      - sql : alter table ${NWIS_SCHEMA_NAME}.network_staging add column if not exists description varchar;
+      - rollback: alter table ${NWIS_SCHEMA_NAME}.network_staging drop column description;
+        
+  - changeSet:
+      author: ajmccart
+      id: "add.contactinfo_column.${NWIS_SCHEMA_NAME}.network_staging"
+      preConditions:
+        - onFail: MARK_RAN
```
Comment (dsteinich) [inline liquibase/changeLogs/nwis/tables/networkStaging/changeLog.yml]: HALT

File: liquibase/changeLogs/nwis/tables/network/changeLog.yml (hunk 1 above)
Comment (ajmccarthy) [reply liquibase/changeLogs/nwis/tables/network/changeLog.yml]: Will do.

**Pull request merged**